README.md

```diff
@@ -1,4 +1,4 @@
-# Kotlinx-Kover
+# kotlinx-kover
```

Comment:
Looked at other kotlinx libraries and it seems like [datetime](https://github.com/Kotlin/kotlinx-datetime), [lincheck](https://github.com/Kotlin/kotlinx-lincheck), [reflect lite](https://github.com/Kotlin/kotlinx.reflect.lite), [cli](https://github.com/Kotlin/kotlinx-cli) and [html](https://github.com/Kotlin/kotlinx.html) all stick to this lower-case artifact name-like style, so changed it for consistency

Alternatively, it could be changed to just `Kover`, it's the `kotlinx` part that's triggering me a bit

Reply:
Perhaps it is better to write `kover gradle plugin`, because in fact this repository contains only a plugin for the Gradle build system, in the future it is possible to create a repository for Maven as well.